index.html

```diff
@@ -1,12 +1,149 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link rel="preconnect" href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+		integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+	<link rel="stylesheet" href="fonts/cocogoose.regular.otf">
+	<link rel="stylesheet" href="styles/styles.css">
 	<title>Bitcoin-Conference 2021 </title>
 </head>
+
 <body>
-	
+	<!-- Header section of the mobile website version starts here -->
+	<header>
+		<nav>
+			<button type="button" class="btn hambuger-menu"><svg xmlns="http://www.w3.org/2000/svg" width="30"
+					height="30" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
+					<path stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="M4 6h16M4 12h16M4 18h16" />
+				</svg></button>
+		</nav>
+		<section class="main-wrapper">
+			<div class="main-nav">
+				<ul id="main-bar">
+					<li><i class="fab fa-facebook-f"></i></li>
+					<li><i class="fab fa-twitter"></i></li>
+					<li><a href="#">Logout</a></li>
+					<li><a href="#">Mypage</a></li>
+					<li><a href="#">English</a></li>
+				</ul>
+			</div>
+			<div class="program-section">
+				<button class="nav__close" id="close-button" type="button">
+					<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor"
+						viewBox="0 0 24 24" stroke="currentColor">
+						<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+							d="M6 18L18 6M6 6l12 12" />
+					</svg>
+				</button>
+				<a href="index.html"> <img src="Assets/Icons/running.jpeg" id="logo" alt="logo"></a>
+				<ul class="nav-menu">
+					<li class="about"><a href="about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsors</a></li>
+					<li><a href="#">News</a></li>
+					<li id="camp"><a href="#">Campaign</a></li>
+				</ul>
+			</div>
```

Comment:
man don't make me throw a fit here, i should personalize it and follow the template. it would have been cool if they just ask me to copy the template verbatim. would white color be consistent with the design if you look at it mate?.